src/headphones2/postprocess/taggers/acoustid_tagger.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+from collections import namedtuple
+from collections import Counter
+from itertools import chain
+
+import acoustid
+import logbook
+import musicbrainzngs
+import py
+
+from headphones2.postprocess.component_base import PostProcessorComponentBase
+
+Path = py.path.local
+
+logger = logbook.Logger(__name__)
+
+API_KEY = '1vOwZtEn'
+SCORE_THRESH = 0.5
+TRACK_ID_WEIGHT = 10.0
+COMMON_REL_THRESH = 0.6  # How many tracks must have an album in common?
+MAX_RECORDINGS = 5
+MAX_RELEASES = 5
+
+
+class AcoustIDAlbumTagger(PostProcessorComponentBase):
+
+    def __init__(self):
+        super(AcoustIDAlbumTagger, self).__init__()
+
+    Result = namedtuple('Result', ['fingerprint', 'acoustid', 'recording_id', 'release_id'])
+
+    @staticmethod
+    def _match_releases(result_list):
+        """
+        determines which releases the items have in common.
+        also uses number of tracks to make sure its the same release
+        """
+        result_ids = chain.from_iterable([result.release_id for result in result_list if result])
+        top_matches = Counter(result_ids).most_common(MAX_RELEASES)
+        num_of_tracks = len(result_list)
+
+        for release_id, _ in top_matches:
+            r = musicbrainzngs.get_release_by_id(release_id, includes='recordings')
+            mediums = r['release']['medium-list']
+            tracks = sum([mediums[i]['track-count'] for i, j in enumerate(mediums)])
+            if tracks == num_of_tracks:
+                return release_id
+
+        return None
+
+    @staticmethod
+    def _acoustid_tag_file(filepath):
+        """
+        Gets metadata for a file from Acoustid.
+        returns a Result object with Fingerprint, acoustid, recording_ids and release_ids
```

Comment:
Should document that any exception is suppressed and None items may be returned.